template.yml

```diff
@@ -22,14 +22,18 @@ Parameters:
     Default: CognitoAuthorizerArn
   UserServiceApiSecretArn:
     Type: 'AWS::SSM::Parameter::Value<String>'
-    Default: UserServiceApiSecretArn
+    Default: /NVA/UserServiceInternalApiKeyArn
```

Comment:
Do we need to prefix the property with NVA? It is all NVA inside the account

Reply:
We probably not NVA, but I think we maybe should prefix them base on some kind of context